relay/polkadot/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Genesis configs presets for the Polkadot runtime
+
+use crate::*;
+use babe_primitives::AuthorityId as BabeId;
+use pallet_staking::{Forcing, StakerStatus};
+use polkadot_primitives::{AccountPublic, AssignmentId, AsyncBackingParams};
+use polkadot_runtime_constants::currency::UNITS as DOT;
+use runtime_parachains::configuration::HostConfiguration;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::{traits::IdentifyAccount, Perbill};
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+use sp_std::vec::Vec;
+
+/// Helper function to generate a crypto pair from seed
+fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Public {
+	TPublic::Pair::from_string(&format!("//{}", seed), None)
+		.expect("static values are valid; qed")
+		.public()
+}
+
+/// Helper function to generate an account ID from seed
+fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId
+where
+	AccountPublic: From<<TPublic::Pair as Pair>::Public>,
+{
+	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
+}
+
+/// Helper function to generate stash, controller and session key from seed
+fn get_authority_keys_from_seed(
+	seed: &str,
+) -> (
+	AccountId,
+	AccountId,
+	BabeId,
+	GrandpaId,
+	ValidatorId,
+	AssignmentId,
+	AuthorityDiscoveryId,
+	BeefyId,
+) {
+	let keys = get_authority_keys_from_seed_no_beefy(seed);
+	(keys.0, keys.1, keys.2, keys.3, keys.4, keys.5, keys.6, get_from_seed::<BeefyId>(seed))
+}
+
+/// Helper function to generate stash, controller and session key from seed
+fn get_authority_keys_from_seed_no_beefy(
```

Comment:
beefy should just be integrated here and have single `get_authority_keys_from_seed` fn

I don't think `get_authority_keys_from_seed_no_beefy` (missing BEEFY) should be still used anywhere

(same for Kusama)

Reply:
ok, `get_authority_keys_from_seed_no_beefy` is removed: https://github.com/polkadot-fellows/runtimes/pull/379/commits/7c05a7a0c186b7883aef41b078d13d61e732a8f2